ExampleMod/Content/Tiles/Furniture/ExampleBed.cs

```diff
@@ -16,16 +18,20 @@ public class ExampleBed : ModTile
 			Main.tileFrameImportant[Type] = true;
 			Main.tileLavaDeath[Type] = true;
 			TileID.Sets.HasOutlines[Type] = true;
-			TileID.Sets.CanBeSleptIn[Type] = true;
+			TileID.Sets.CanBeSleptIn[Type] = true; // Facilitates calling ModifySleepingTargetInfo
+			TileID.Sets.InteractibleByNPCs[Type] = true;
 			TileID.Sets.IsValidSpawnPoint[Type] = true;
 			TileID.Sets.DisableSmartCursor[Type] = true;
 
+			AddToArray(ref TileID.Sets.RoomNeeds.CountsAsChair);
```

Comment:
We need a comment here clarifying why beds count as chairs I think. InteractibleByNPCs also